README.md

```diff
@@ -22,7 +32,7 @@ These are the machines and how they are connected with each other:
 <img src="diagram.png">
 
 
-# Requirements
+# Install
 
 `virtualbox` and `vagrant` and `virtualbox-extension-pack` are the dependencies for this repository.
```

Comment:
```suggestion
`vagrant`, `virtualbox`, and `virtualbox-extension-pack` are required dependencies for this repository.
```

Reply:
I originally put a removed the `and`s in between the required installs but the `and` is more readable and separates better in the render Markdown.